docs/README.md

```diff
@@ -75,6 +108,229 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+      version: v1.21.0
+    EOF
+    ```
+
+3. Confirm the installation and version of the control plane.
+
+    ```bash
+    $ kubectl get istio -n istio-system
+    NAME      READY   STATUS    IN USE   VERSION   AGE
+    default   True    Healthy   True     v1.21.0   2m
+    ```
+
+4. Create namespace `bookinfo` and deploy bookinfo application
+
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo istio-injection=enabled
+    kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml
+    ```
+Note: if the `Istio` resource name is other than default, you need to label the namespace with the revision name.
+
+5. Perform the update of the control plane by changing the version in the Istio resource
+
+    ```bash
+    kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.21.2"}}'
+    ```
+
+6. Confirm the istio resource version was updated
+
+    ```bash
+    $ kubectl get istio -n istio-system
+    NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   VERSION   AGE
+    default   1           1       1        Healthy           v1.21.2   12m
+    ```
+
+7. Delete `bookinfo` pod to trigger sidecar injection with the new version
+
+    ```bash
+    kubectl rollout restart deployment -n bookinfo
+    ```
+
+8. Confirm that the new version is used in the sidecar
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should match the new control plane version.
+
+### RevisionBased
+When the `RevisionBased` strategy is used, a new Istio control plane instance is created for every change to the `Istio.spec.version` field. The old control plane remains in place until all workloads have been moved to the new control plane instance, then, the old control plane is deleted.
+
+#### Example using the RevisionBased strategy
+
+Prerequisites:
+* sail-operator needs to be installed.
+* Deploy `IstioCni` resource.
+* `istioctl` is installed.
+
+Steps:
+
+1. Create the `istio-system` namespace.
+
+    ```bash
+    kubectl create namespace istio-system
+    ```
+
+2. Create the `Istio` resource
+
+    ```bash
+    cat <<EOF | kubectl apply -f-
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      namespace: istio-system
+      updateStrategy:
+        type: InPlace
+        inactiveRevisionDeletionGracePeriodSeconds: 30
+      version: v1.21.0
+    EOF
+    ```
+
+3. Confirm the control plane is installed and is using the desired version.
+
+    ```bash
+    $ kubectl get istio -n istio-system
+    NAME      READY   STATUS    IN USE   VERSION   AGE
+    default   True    Healthy   True     v1.21.0   2m
+    ```
+
+4. Get the `istiorevision` name
+
+    ```bash
+    $ kubectl get istiorevision -n istio-system
+    NAME              READY   STATUS    IN USE   VERSION   AGE
+    default-v1-21-0   True    Healthy   False    v1.21.0   114s
+    ```
```

Comment:
Maybe add note saying that the revision name is composed of the `Istio` resource name and the version.